tests/ui/allow_attributes_false_positive.rs

```diff
@@ -0,0 +1,6 @@
+#![allow(unused)]
```

Comment:
Thanks! For clarity lets remove this, I didn't realise it only linted on outer attrs so I was wondering why this wasn't linted

```suggestion
```